landingzones/views.py

```diff
@@ -220,6 +221,45 @@ def submit_create(
             zone.delete()
             raise ex
 
+    def update_zone(self, zone, request=None):
+        """
+        Handle timeline updating after a LandingZone object has been updated.
+        :param zone: LandingZone object
+        :param form: LandingZoneForm object
+        :raise: taskflow.FlowSubmitException if taskflow submit fails
+        """
+
+        timeline = get_backend_api('timeline_backend')
+        user = request.user if request else None
+
+        # Add event in Timeline
+        if timeline:
+            tl_extra = {
+                'title': zone.title,
+                'assay': str(zone.assay.sodar_uuid),
+                'description': zone.description,
+                'user_message': zone.user_message,
+            }
+            tl_event = timeline.add_event(
+                project=zone.project,
+                app_name=APP_NAME,
+                user=user,
+                event_name='zone_update',
+                description='update landing zone {{{}}} for {{{}}} in '
+                '{{{}}}'.format('zone', 'user', 'assay'),
```

Comment:
Since `zone`, `user` and `assay` are all strings and not variables, we can simplify this into:
```
description='update landing zone {zone} for {user} in {assay}',
```